src/tests/functional/shared_test_classes/src/base/layer_test_utils.cpp

```diff
@@ -332,6 +332,45 @@ void LayerTestsCommon::Compare(const InferenceEngine::Blob::Ptr &expected, const
     }
 }
 
+void LayerTestsCommon::Compare(const ov::Tensor &expected, const ov::Tensor &actual) {
```

Comment:
This class is based on API1.0. strongly recommend to deprecate it instead of refactoring. `SubgraphBaseTest` is whole analog of the class based on API2.0

Reply:
Ok. I will change it to SubgraphBaseTest